definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -6298,5 +6298,54 @@
     "CaseEventID": "amendPetitionForRefusalRejection",
     "UserRole": "caseworker-divorce-courtadmin-la",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "23/05/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "createCase",
+    "UserRole": "caseworker-divorce-systemupdate",
```

Comment:
This one isn't necessary, because Bulk Scan will use this event authenticated as the caseworker who triggered the action.